owncloud/core
# Revert "Move user creation to a separate service"

This reverts commit 30f0a5e7f26211b9b30dbbe5363769efaf9c81fa.



## Description

No sufficient time to stabilize for 10.3. Hence we need to revert the change.

## Related Issue




- Fixes <issue_link>

## Motivation and Context

There are inconsistencies with the messages written in the console. And hence there is not sufficient time to stabilize it before 10.3. Hence reverting the change.

## How Has This Been Tested?



- After reverting the change:
- Create user command line:
```
sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ●  ./occ user:add --display-name "Test User1" --email "********" --group "group1" --group "group2" user1
Cannot load Xdebug - it was already loaded
Enter password: 
Confirm password: 
The user "user1" was created successfully
Display name set to "Test User1"
Email address set to "********"
Created group "group1"
User "user1" added to group "group1"
Created group "group2"
User "user1" added to group "group2"
 sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ● 
```
- Create user using provision api:
```
sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ●  curl -X POST http://admin:admin@localhost/testing3/ocs/v1.php/cloud/users -d userid="test2" -d email='*****' -d groups\[\]="group1" -d groups\[\]="group2" -d password="1"
<?xml version="1.0"?>
<ocs>
 <meta>
  <status>ok</status>
  <statuscode>100</statuscode>
  <message/>
 </meta>
 <data/>
</ocs>
 sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ● 
```
- Verify that users are available:
```
sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ●  ./occ user:list                                                                                                   
Cannot load Xdebug - it was already loaded
  - admin: admin
  - test2: test2
  - user1: Test User1
 sujith@sujith-ownCloud  ~/test/owncloud3   revert-pr-35972 ●  
```

## Screenshots (if appropriate):

## Types of changes


## Checklist:



## Open tasks:
